sponsorship.md

```diff
@@ -0,0 +1,46 @@
+# Pangeo Sponsorship Policy
+
+Now that Pangeo has a 501c3 fiscal sponsor, it can receive donations. Individuals or institutions who make donations to the project are **Project Sponsors**. The purpose of this document is to define the relationship between the Project and Project Sponsors.
+
+## Types of Sponsorship
+
+There are two ways to sponsor the Project:
+1. Direct financial contribution - a direct donation made via [NumFocus](https://numfocus.org/donate).
+2. In-kind contribution - goods or services provided to the project in support of its activities. Examples of in-kind contributions include:
+
+   - Personnel time dedicated to project support and administration. This excludes work on the project by Steering Council members.
+   - Computers, cloud credits, or other IT-related services paid for by the sponsor on behalf of the project.
+   - Access to facilities such as office space, meeting space, or shared conference booths.
+
+  When making an in-kind donation, the sponsor should notify NumFocus of the donation and quantify the value of the in-kind donation. (For donors in the United States, such donations are tax-deductible to the extent provided by law. As with any donation, donors should consult with their tax adviser about their particular tax situation.)
+
+The Steering Council or NumFOCUS may reject sponsorship offers that do not align with the Project values or mission.
+
+## Relationship between the Project and Sponsors
+
+The sponsorship relationship should be mutually beneficial to the Project and the Sponsor. The benefits of sponsorship to the Project are straightforward: more resources with which to achieve goals identified in the Project Roadmap and Project Mission. Some of the benefits to the Sponsor are:
```

Comment:
```suggestion
The sponsor-project relationship should be mutually beneficial to the Project and the Sponsor. The benefits of sponsorship to the Project are straightforward: more resources to achieve goals identified in the [roadmap](https://pangeo.io/roadmap.html) and [mission](https://pangeo.io/about.html#mission-statement). Some of the benefits to the Sponsor are:
```

I'm also wondering if "the Sponsor" should be capitalized everywhere, and perhaps replaced with "sponsors", since we're not talking about any identified sponsor.